### ISSUE TITLE
Change title of treemap

### ISSUE THREAD

**Issue by yannickvandendijck** [openanalytics/clinDataReview#4]
How can the 'title' be changed of a treemap? Where/how should this be specified in the yml-code?

countVar: "ARM"
parentVar: ["COUNTRY","SITEID"]
typePlot: [treemap]

**Comment by lcougnaud** [comment]
You are right, this is not yet possible to specify the title if the treemap is created in a medical monitoring report. This will be updated it in the package.
